src/deadline/unreal_submitter/unreal_logger/logger.py

```diff
@@ -0,0 +1,33 @@
+import os
+import json
+import logging
+import logging.config
+from typing import Optional
+
+from deadline.unreal_submitter.unreal_logger.handlers import UnrealLogHandler
+
+
+with open(f"{os.path.dirname(__file__)}/config.json", "r") as f:
+    LOGGER_CONFIG = json.load(f)
+
+
+def get_logger() -> logging.Logger:
+
+    logging.config.dictConfig(LOGGER_CONFIG)
+
+    unreal_logger = logging.getLogger("unreal_logger")
+
+    unreal_log_handler: Optional[UnrealLogHandler] = next(
```

Comment:
What are we doing in here?  We'll want to comment this at least if it is all necessary

Reply:
Since unreal can handle both stdout and unreal.log messages we can get log duplication for each logger.info() call. So as default behaviour - we remove unreal handler if its not initialized properly and use default stdout. And vice versa, we delete the regular one and leave the unreal one

Reply:
Ahh I think I see - our config file has both handlers declared.  Yeah the other options would be to have those live in different files that we used depending on which we were trying to initialize.  Or yeah I guess the question of data vs code - code might make that a little easier to just add the one thing we want.  It just seems a bit odd to be adding two loggers then conditionally removing one rather than just add the one we want.  But yes if we do keep things this way we'll want to comment what's going on because it looks a bit unusual